src/AccordionSection/StyledAccordionPanel.js

```diff
@@ -1,9 +1,10 @@
 import styled from '@emotion/styled';
 
 import Box from '../Box';
-import createWithComponent from '../utils/createWithComponent';
 
-const StyledAccordionPanel = styled(createWithComponent(Box, 'div'))`
+const StyledAccordionPanel = styled(Box)`
+  border-bottom: 1px solid ${props => props.theme.borderColors.default};
```

Comment:
This is getting passed in as a prop on `AccordionPanel` - https://github.com/CatchRelease/arbor/blob/master/src/AccordionSection/AccordionPanel.js#L11

So we may not need that here. Though, I guess that makes things weird with the `last-child` below.

Reply:
I think we could remove it here. This way you could pass in a different option but still hide it from the last child.

Reply:
Yeah, the only thing is we’re applying a border-top in the case of the last child to. 🤔